### PR TITLE
Correct grammatical and spelling errors in sample documentation

### LINKED ISSUE #29402: Sample docs contain typos and grammar issues
`concepts/Sample.html` contains several typos, full stops in place of colons, etc

### PR DESCRIPTION
**Pull request by Harrietbrown**
This PR corrects spelling and grammar errors in the sample documentation.

re: #29402

**Description of work.**

This PR corrects spelling and grammar errors in the sample documentation.

**To test:**

Build docks and check that `concepts/Sample` does not contain grammatical errors.

Fixes #29402

*This does not require release notes* because It is only a change to spelling in documentation

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/concepts/Sample.rst [hunk 1]
```diff
@@ -73,7 +75,7 @@ Sample Properties
 Multiple Samples
 ################
 
-The ``Sample()`` method actually returns a collection, however if you do not specify which sample you are after you will get he first member of the collection. So 
+The ``Sample()`` method actually returns a collection, however if you do not specify which sample you are after you will get the first member of the collection.
```
Comment (robertapplin) [inline docs/source/concepts/Sample.rst:78]: I think there might be a typo in this sentence? 'however if you do not specify which sample you are *using* ~~after~~ you will get the first member of the collection'

**Review by robertapplin — CHANGES_REQUESTED** [review]
These changes look good, just one typo(?) remaining

File: docs/source/concepts/Sample.rst (hunk 1 above)
Comment (Harrietbrown) [reply docs/source/concepts/Sample.rst:78]: Could be better phrased for non native english speakers. it doesn't read very easy. to say "what you are after" meaning "what you want"

**Review by robertapplin — DISMISSED** [review]

**Comment by clayton-tom** [comment]
If this is for 5.1 do you need to rebase onto `release-next` and change the merge target? (Genuine question, I may have misunderstood!)

**Review by robertapplin — APPROVED** [review]
Good changes to the Sample docs 👍

**Pull request merged**